stacks/dotnetcore31/devfile.yaml

```diff
@@ -35,7 +35,8 @@ components:
         - name: ASPNETCORE_URLS
           value: http://*:8080
       endpoints:
-        - name: http-dotnetcore
+        - name: https-dotnet
```

Comment:
Sound better to me if it remains `https-dotnetcore`. WDYT?

Reply:
Oh I see! Then I think is fine to leave it as is

Reply:
Endpoint name `https-dotnetcore` appears to be too long. 
We have already discussed it here https://github.com/devfile/registry/pull/398#discussion_r1605008509

Anyway, `https-dotnetcore` much better suits here, I wonder whether it possible to increase the maximum length of the name somewhere?

Reply:
Is mentioned [here](https://devfile.io/devfile-schemas/2.2.2.json) in the schema. IMO is better if we keep the current name and don't change the devfile schema.